frontend/src/employee-frontend/utils/money.ts

```diff
@@ -2,15 +2,20 @@
 //
 // SPDX-License-Identifier: LGPL-2.1-or-later
 
-export function formatCents(amount?: number): string | undefined {
+export function formatCents(
+  amount?: number,
+  fixedCents?: boolean
+): string | undefined {
   if (amount === undefined || amount === null) {
     return undefined
   }
 
   const euros = amount >= 0 ? Math.floor(amount / 100) : Math.ceil(amount / 100)
   const cents = Math.abs(amount % 100)
 
-  return `${euros}${cents !== 0 ? `,${cents.toString().padStart(2, '0')}` : ''}`
+  return `${euros}${
+    cents !== 0 || fixedCents ? `,${cents.toString().padStart(2, '0')}` : ''
```

Comment:
Should this maybe be split into multiple parts? It's kind of hard to read what this function produces.